packages/smart/smart_p.go

```diff
@@ -39,21 +39,14 @@ import (
 
 var (
 	funcCallsDBP = map[string]struct{}{
-		"DBInsert":         {},
-		"DBUpdate":         {},
+		"DBInsert":       {},
+		"DBUpdate":       {},
 		"DBUpdateSysParam": {},
-		"DBUpdateExt":      {},
-		"DBSelect":         {},
-		"DBInt":            {},
-		"DBRowExt":         {},
-		"DBRow":            {},
-		"DBStringExt":      {},
-		"DBIntExt":         {},
-		"DBStringWhere":    {},
-		"DBIntWhere":       {},
-		"DBAmount":         {},
-		"DBInsertReport":   {},
-		"FindEcosystem":    {},
+		"DBUpdateExt":    {},
+		"DBSelect":       {},
+		"DBAmount":       {},
```

Comment:
Are you sure that DBAmount func is good idea? SELECT amount FROM __table__, i think it is better to delete it too, and use DBRow(), than special convertion function to Decimal